ZWaveLib/Values/CentralSceneValue.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ZWaveLib.Values
+{
+    public enum CentralScenePressType
+    {
+        Pressed1times = 0x0,
+        Released = 0x1,
+        HeldDown = 0x2,
+        Pressed2times = 0x3,
+        Pressed3times = 0x04,
+        Pressed4times = 0x05,
+        Pressed5times = 0x06,
+        Reserved = 0x07,
+    }
+
+    public enum CentralSceneValueID
+    {
+        SceneCount = 0x00,
+        Scene_KeyAttribute = 0x01,
+        SceneID = 0x02,
+        Button = 0x03,
+        Scenes_Identical = 0x04,
+        Supported_KeyAttributes_All_Scenes = 0x05,
+        Supported_KeyAttributes_Scene_1 = 0x06,
+        Supported_KeyAttributes_Scene_2 = 0x07,
+        Supported_KeyAttributes_Scene_3 = 0x08,
+        SceneNumber = 0x80,
+    }
+
+    public class CentralSceneValue
+    {
+        // Layout
+        // 1 = MessageType
+        // 2 = Level- up from 0x75
+        // 3 = KeyAttribute
+        // 4 = Scene ID
+        public EventParameter EventType = EventParameter.CentralScene;
+        public byte SceneId; // byte 4
+        public byte Level; // byte 2 
+        public CentralScenePressType PressType; // byte 3
+
+        public static CentralSceneValue Parse(byte[] message)
+        {
+            int keyAttribute = message[3];
+
+            // Hack for Fibaro Keyfob
+            if (keyAttribute > 0x80)
+                keyAttribute -= 0x80;
```

Comment:
This looks strange.
I think you have found that Fibaro Keyfob implements Central Scene Command Class V3, while you are implementing a mix of Central Scene Command Class V1 and V2.
In V3 there was introduced Slow Refresh flag in Central Scene Notification Command (see SDS13781-2 paragraph 4.17.7) which is the most significant bit, that's why you are subtracting the value of 0x80.
I think you should read Key Attribute from 3 least significant bits only.